stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/style/ChatStyleImpl.kt

```diff
@@ -0,0 +1,9 @@
+package com.getstream.sdk.chat.style
+
+internal class ChatStyleImpl : ChatStyle {
```

Comment:
Could we move the logic of this class into the interface?
I don't really see needed `ChatStyle` as interface, it could be an open class

Reply:
The only pro is it can be internal if it's exposed by the interface. Otherwise, I'll need to make it public.